docs/contributing/local-testing.md

```diff
@@ -0,0 +1,186 @@
+# Local testing of Claudie
+
+In order to speed up the development, Claudie can be run locally for initial testing purposes. However, it's important to note that running Claudie locally has limitations compared to running it in a Kubernetes cluster.
+
+## Limitations of Claudie when running locally
+
+### Frontend/CRD testing
+
+The Frontend component as well as CRDs heavily relies on the Kubernetes cluster. However, with a little hacking, you can test them, by creating local cluster (minikube/kind/...), and exporting environment variable `KUBECONFIG` pointing to the local cluster Kubeconfig. Once you start the Frontend, it should pick up the Kubeconfig and you can use local cluster to deploy and test CRDs.
```

Comment:
Just one point from my end. 
If I recall correctly, when I had altered the source code I had to build new images, copy them to kind/minikube/... cluster, change the images tags in [/manifests/claudie/kustomization.yaml](https://github.com/berops/claudie/blob/master/manifests/claudie/kustomization.yaml) and then deploy Claudie in kind/minikube/... cluster. 
What do you think about creating a small section describing this process?

Reply:
Have a look at the latest commit, and let me know if this is enough info about local cluster testing